cli/src/commands/args.rs

```diff
@@ -510,6 +510,10 @@ pub struct TunnelServeArgs {
 	/// Randomly name machine for port forwarding service
 	#[clap(long)]
 	pub random_name: bool,
+
+	/// Optional parent process id. If provided, the server will be stopped when the process of the given pid no longer exists
+	#[clap(long)]
```

Comment:
We can add this to hide it from `--help` since I think most users don't need it...

```suggestion
	#[clap(long, hide = true)]
```